mongodbatlas/data_source_mongodbatlas_online_archive.go

```diff
@@ -0,0 +1,212 @@
+package mongodbatlas
+
+import (
```

Comment:
[q] Why don't we have a test file for `data_source_mongodbatlas_online_archive.go` and `resource_mongodbatlas_private_endpoint.go` ? Is it because they need a cluster up and running? if yes, Don't we have unit tests? Thanks!!

Reply:
Hello @andreaangiolillo, in the provider most of the tests are [Acceptance Tests](https://www.terraform.io/docs/extend/testing/acceptance-tests/index.html) to actually create the infra through the go client (mongodbatlas). For this case the limitation is that I need the cluster with data already preseeded . We could automate it like create the cluster, add another client to create the collection and insert some dummy data, but that would mean that just this test could take like ~30 min and I will need to include extra dependencies not intended for the provider itself. :)